ipv-stub/src/interfaces/reverification-interface.ts

```diff
@@ -1,6 +1,6 @@
 export interface Reverification {
   sub: string;
   success: boolean;
-  error_code?: string;
-  error_description?: boolean;
+  failure_code?: string;
+  failure_description?: string;
 }
```

Comment:
Not necessary for the stub, but if you wanted to get super fancy (and ensure `error_code` can't exist without an `error_description`) you could do:

```
export interface ReverificationSuccess {
  sub: string;
  success: true;
}

export interface ReverificationFailure {
  sub: string;
  success: false;
  error_code: string;
  error_description: boolean;
}

export type Reverification = ReverificationSuccess | ReverificationFailure;
```

Reply:
I'll chuck that in, I'm happy to be super fancy